### PR TITLE
LPS-142527 Maintain selected vocabularies when moving them from one box to another

### PR DESCRIPTION
**Pull request by claraizquierdo**
Motivation
=======
When moving from 'Available' to 'In Use', selected status it's lost.
[Link to story or ticket](https://issues.liferay.com/browse/LPS-142527)

Proposed Solution
========
We needed to update the selected vocabularies in each box after moving a vocabulary from left to right or right to left.

### CODE REVIEW

**Comment by claraizquierdo** [comment]
ci:test:sf

**Comment by claraizquierdo** [comment]
ci:test:relevant

**Review by beltranrengifo — CHANGES_REQUESTED** [review]
Hi @claraizquierdo !
Just a minor comment here

File: modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/VocabulariesSelectionBox.js [hunk 1]
```diff
@@ -125,6 +130,35 @@ const VocabulariesSelectionBox = ({
 		);
 	};
 
+	const maintainSelectedVocabularies = () => {
+		const previousLeftElements = previousLeftElementsRef.current;
+
+		if (
+			typeof previousLeftElements !== 'undefined' &&
```
[the remark this answers is not in the document]
Comment (claraizquierdo) [reply modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/VocabulariesSelectionBox.js:137]: Good point Beltran 👍

**Comment by claraizquierdo** [comment]
ci:test:sf

**Review by beltranrengifo — APPROVED** [review]
🎉

**Comment by claraizquierdo** [comment]
ci:forward

**Comment by claraizquierdo** [comment]
ci:forward

**Comment by claraizquierdo** [comment]
ci:forward

**Comment by claraizquierdo** [comment]
ci:test:relevant

**Comment by claraizquierdo** [comment]
ci:test:tango

**Comment by claraizquierdo** [comment]
ci:forward

**Pull request closed without merging**